platform/darwin/src/MGLAttributedExpression.h

```diff
@@ -19,9 +32,21 @@ MGL_EXPORT
  */
 @property (strong, nonatomic) NSExpression *expression;
 
+#if TARGET_OS_IPHONE
+/**
+ The formatting attributes dictionary.
+ `MGLFontNamesAttribute` : `NSArray<NSString *>*`
+ `MGLFontSizeAttribute` : `NSNumber`
+ `MGLFontColorAttribute` : `UIColor`
```

Comment:
This probably won’t format how you’d expect it to in the published docset. Use a Markdown or HTML table for this information. While you’re at it, the “Predicates and Expressions” guide should include this table so the types are known, and “Information for Style Authors” should list a correspondence between the attributes in the style specification and these constants.